opennms-webapp-rest/src/main/java/org/opennms/web/rest/v1/MonitoringLocationsRestService.java

```diff
@@ -117,19 +116,6 @@ public Response addMonitoringLocation(@Context final UriInfo uriInfo, OnmsMonito
 		try {
 			LOG.debug("addMonitoringLocation: Adding monitoringLocation {}", monitoringLocation.getLocationName());
 			m_monitoringLocationDao.save(monitoringLocation);
-
-			final boolean sendEvent = monitoringLocation.getPollingPackageNames() != null && !monitoringLocation.getPollingPackageNames().isEmpty();
```

Comment:
The POLLER_PACKAGE_LOCATION_ASSOCIATION_CHANGED_EVENT_UEI can be removed, but it's handling must be moved to the ApplicationRestService. We already have an APPLICATION_DELETED_EVENT_UEI, so I think we should introduce an additional APPLICATION_CHANGED_EVENT_UEI in order to re-schedule services at runtime.

Reply:
Same for an `APPLICATION_CREATED_EVENT_UEI`

Reply:
See https://issues.opennms.org/browse/NMS-12838